src/content/getusermedia/unprocessed-volume/js/main.js

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* global AudioContext, SoundMeter */
+
+'use strict';
+
+var instantMeter = document.querySelector('#instant meter');
+var slowMeter = document.querySelector('#slow meter');
+var unprocessedMeter = document.querySelector('#unprocessed meter');
+
+var instantValueDisplay = document.querySelector('#instant .value');
+var slowValueDisplay = document.querySelector('#slow .value');
+var unprocessedValueDisplay = document.querySelector('#unprocessed .value');
+
+try {
+  window.AudioContext = window.AudioContext || window.webkitAudioContext;
+  window.audioContext = new AudioContext();
+} catch (e) {
+  alert('Web Audio API not supported.');
+}
+
+// Put variables in global scope to make them available to the browser console.
+var constraints = window.constraints = {
+  audio: {echoCancellation: true},
+  video: false
+};
+
+function handleSuccess(stream) {
+  // Put variables in global scope to make them available to the
+  // browser console.
+  window.stream = stream;
+  var soundMeter = window.soundMeter = new SoundMeter(window.audioContext);
+  soundMeter.connectToSource(stream, function(e) {
+    if (e) {
```

Comment:
You could expand `e` to error since you have space. Also not sure if alert is preferred anymore but not outputting text on the page might not be clear enough.